go.mod

```diff
@@ -1,17 +1,41 @@
 module github.com/snyk/snyk-iac-rules
 
-go 1.16
+go 1.17
 
 require (
 	github.com/cheggaaa/pb/v3 v3.0.8
 	github.com/ghodss/yaml v1.0.0
-	github.com/hashicorp/hcl/v2 v2.10.1 // indirect
 	github.com/open-policy-agent/opa v0.32.0
-	github.com/rcrowley/go-metrics v0.0.0-20201227073835-cf1acfcdf475 // indirect
 	github.com/spf13/cobra v1.2.1
 	github.com/stretchr/testify v1.7.0
 	github.com/tmccombs/hcl2json v0.3.1
+)
+
+require (
```

Comment:
Is the larger indirect list an artifact of the 1.17 bump? Did you run `go mod tidy`?

I really recommend adding something like `go mod tidy && git diff --exit-code` to CI to assert that the module list is tidy as a pre-merge check. We've got this in policy-engine, but in circle, so you'd have to adapt that to github actions if you're using that here. Perhaps there's a marketplace action that does this?